src/LearnositySdk/Request/Init.php

```diff
@@ -140,6 +150,50 @@ public function __construct($service, $securityPacket, $secret, $requestPacket =
         $this->securityPacket['signature'] = $this->generateSignature();
     }
 
+    /**
+     * Adds metadata to request packet to enable telemetry and tracking. Request packet will be
+     * extended with following parameters:
+     * {
+     *   "meta": {
+     *     "sdk": {
+     *       "version": "v0.10.0",
+     *       "lang": "php",
+     *       "lang_version": "5.6.36",
+     *       "platform": "Linux",
+     *       "platform_version": "3.10.0-862.6.3.el7.x86_64"
+     *     }
+     *   }
+     * }
+     *
+     * @param array    $requestPacket
+     *
+     * @return array
+     */
+    private function addMeta(array $requestPacket) {
+        $requestPacket['meta'] = [
+            'sdk' => [
+                'version' => $this->getSDKVersion(),
+                'lang' => 'php',
+                'lang_version' => phpversion(),
+                'platform' => php_uname('s'),
+                'platform_version' => php_uname('r')
+            ]
+        ];
+
+        return $requestPacket;
+    }
+
+    private function getSDKVersion() {
+        $rootDir = __DIR__ . '/../../..';
+        $versionFilePath = $rootDir . '/.version';
```

Comment:
I'm late to the party... Sorry. Can this be a `const` at the class level? I don't think we need the `$rootDir` to be separate.

Reply:
No worries - still not too late ;) make sense and may do... I remember we had a bit of a chat about it already hehe